cvat-ui/src/actions/tasks-actions.ts

```diff
@@ -414,7 +414,7 @@ export function createTaskAsync(data: any): ThunkAction<Promise<void>, {}, {}, A
         dispatch(createTask());
         try {
             const savedTask = await taskInstance.save((status: string, progress: number): void => {
-                dispatch(createTaskUpdateStatus(status + (progress !== null ? ` ${Math.floor(progress * 100)}%` : '')));
+                dispatch(createTaskUpdateStatus(status + (progress ? ` ${Math.floor(progress * 100)}%` : '')));
```

Comment:
If the progress is 0, it does not show any progress. Maybe to keep ``!== null`` check?

Reply:
Ok, lets pass `null` if no progress has to be specified